src/XrdHttp/XrdHttpReq.cc

```diff
@@ -1112,40 +1133,38 @@ int XrdHttpReq::ProcessHTTPReq() {
                   if (prot->staticpreload) {
                     XrdHttpProtocol::StaticPreloadInfo *mydata = prot->staticpreload->Find(resource.c_str());
                     if (mydata) {
+                      logTransferEvent(TRACE_STATSALL, 200, "STATIC_DATA_OK");
                       prot->SendSimpleResp(200, NULL, NULL, (char *) mydata->data, mydata->len, keepalive);
                       reset();
                       return keepalive ? 1 : -1;
                     }
                   }
-                  
                 }
-
-
           }
-      
+
       switch (reqstate) {
         case 0: // Stat()
-          
           // Do a Stat
           if (prot->doStat((char *) resourceplusopaque.c_str())) {
             XrdOucString errmsg = "Error stating";
             errmsg += resource.c_str();
+            logTransferEvent(TRACE_STATSALL, 404,(char *) errmsg.c_str());
```

Comment:
Why the const-cast here?  That suggests maybe the definition of `logTransferEvent` is wrong?